### PR TITLE
ci: add k8s-pid-ns.bats

### PR DESCRIPTION
**Pull request by Tim-0731-Hzt**
add k8s-pid-ns.bats

Fixes:#5335
Signed-off-by: Zhongtao Hu <zhongtaohu.tim@linux.alibaba.com>

### CODE REVIEW

File: integration/kubernetes/run_kubernetes_tests.sh [hunk 1]
```diff
@@ -95,6 +95,7 @@ else
                 "k8s-shared-volume.bats" \
                 "k8s-volume.bats" \
                 "k8s-nginx-connectivity.bats" \
+				"k8s-pid-ns.bats" \
```
Comment (studychao) [inline integration/kubernetes/run_kubernetes_tests.sh:98]: Seems like there are some extra spaces.

**Review by studychao — COMMENTED** [review]
Thanks, few comments.

File: integration/kubernetes/run_kubernetes_tests.sh (hunk 1 above)
Comment (Tim-0731-Hzt) [reply integration/kubernetes/run_kubernetes_tests.sh]: fixed

**Review by fidencio — APPROVED** [review]
lgtm, thanks @Tim-0731-Hzt!

**Comment by fidencio** [comment]
@Tim-0731-Hzt, mind to rebase this PR due to the conflicts?

**Review by GabyCT — APPROVED** [review]

**Pull request merged**